downstream/assemblies/eda/assembly-eda-set-up-rhaap-credential.adoc

```diff
@@ -0,0 +1,7 @@
+[id="eda-set-up-rhaap-credential-type"]
+
+= {PlatformName} credential
+
+To run your rulebook activations successfully, you must create credentials with the {PlatformName} credential type. Using this credential type provides more seamless authentication without the requirement to create a token.
```

Comment:
@jself-sudoku We still need a token unless the user chooses user/password, so can the second sentence mention the previous "Controller Token" in EDA UI.
```
Using this credential type provides more seamless authentication without the requirement to create the "Controller Token" in the prior version of the product.
```

Reply:
ok

Reply:
@mkanoor - this is a little confusing. I thought the "Controller Token" message and procedural link had been removed from the UI and was being replaced by this method.

Reply:
Thank you for the explanation, Madhu. I will remove this portion and refresh this PR.

Reply:
@mkanoor - I made some changes to the content, based on some information you shared in one of your Google docs related to credentials and credential types. Since @zkayyali812 approved this PR earlier, I think we should be able to push this forward in our process. Can you take a look at what I have? In the meantime, I've moved this to peer review so that we can publish to our staged docs. My teammate needs the link to add to a KCS article. If we need to tweak the content later, we can. I just don't want to hold this content back any longer. Thanks for helping me gain better understanding on RH AAP credentials.